Userland/DynamicLoader/CMakeLists.txt

```diff
@@ -21,3 +21,23 @@ target_link_options(Loader.so PRIVATE LINKER:--no-dynamic-linker)
 target_link_libraries(Loader.so PRIVATE NoCoverage)
 
 install(TARGETS Loader.so RUNTIME DESTINATION usr/lib/)
+
+## Build an auto-jailing version of Loader.so
+
+add_library(JailedDynamicLoader_CompileOptions INTERFACE)
```

Comment:
I think you can just use `DynamicLoader_CompilerOptions` with `target_compile_definitions(ldjail.so PRIVATE ENTER_JAIL)` (also, no need to carry `_` prefix)